colavo/lib/email/resend-service.ts

```
@@ -0,0 +1,89 @@
+   */
+  static async updateEmail(emailId: string, newScheduledAt: Date): Promise<void> {
+    try {
+      await resend.emails.update({
+        id: emailId,
+        scheduledAt: newScheduledAt.toISOString(),
+      });
+    } catch (error) {
+      throw new Error(`Failed to update email schedule: ${error instanceof Error ? error.message : 'Unknown error'}`);
+    }
+  }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)